conf/messages

```diff
@@ -207,5 +208,6 @@ inspection_needed_export.ogd.heading=For other government department inspections
 inspection_needed_export.ata.heading=For your ATA Carnet endorsement
 inspection_needed_export.tir.heading=For your TIR Carnet endorsement
 inspection_needed_export.sad.heading=For your Single Administrative Document (SAD) endorsement
+inspection_needed_export.sns.heading=For your ENS (Safety and Security) inspection
```

Comment:
this will never be called, if in the case it is called, saying ENS for export is not right.

Reply:
Hard to know what's best here, as although we shouldn't ever get inspectionTypeId 16 back for exports, the logic can't be coded based on hard assumptions of which ids we get back for which directions. 

Wonder if it's best to take the key out altogether; in the highly unlikely event we receive that inspectionTypeId back for exports, it would presumably default to showing the missing message key on screen, which would at least make the issue clear to us quickly.

Reply:
yeah, thats the reason why i wanted to add these export keys incase we get have to add inspectionTypeIds for exports in future.. but i can delete them now if you guys want to delete

Reply:
deleted